ws-nextjs-app/.gitignore

```diff
@@ -40,4 +40,5 @@ next-env.d.ts
 .vscode
 
 # Consider removing this once we move Express app over to Next.js
-/public
\ No newline at end of file
+/public
+certificates
```

Comment:
Is this generated from the experimental https flag?

Reply:
Yeah it generates self signed certificates for localhost https://vercel.com/guides/access-nextjs-localhost-https-certificate-self-signed